k8s/aat/common/ccd/definition-store-api.yaml

```diff
@@ -28,6 +28,7 @@ spec:
       environment:
         ELASTIC_SEARCH_FAIL_ON_IMPORT: true
         DEFINITION_STORE_DB_MAX_POOL_SIZE: 25
+        DEFINITION_STORE_S2S_AUTHORISED_SERVICES: em_hrs_api
```

Comment:
this will break aat env. We need to make sure we are appending the service id to the allowed-listed ids inherent from chart config.